datahub-frontend/build.gradle

```diff
@@ -80,6 +80,7 @@ docker {
     include 'docker/monitoring/*'
     include "docker/${docker_dir}/*"
   }
+  tag("Debug", "${docker_registry}/${docker_repo}:debug")
```

Comment:
So this publishes a debug container... When the release tag === Debug? or always?

Reply:
Never publishes (pushes), just builds it locally.